UnityProject/Assets/Scripts/HealthV2/Living/MedicalChemistry/BodyHealthEffect.cs

```diff
@@ -56,14 +56,14 @@ public override void PossibleReaction(BodyPart sender, ReagentMix reagentMix, fl
 				{
 					foreach (var Effect in Effects)
 					{
-						sender.TakeDamage(null, Effect.EffectPerOne * 3 * LimitedreactionAmount * -OverdoseDamageMultiplier, Effect.AttackType,
+						sender.TakeDamage(null, Effect.EffectPerOne * 6 * LimitedreactionAmount * -OverdoseDamageMultiplier, Effect.AttackType,
```

Comment:
Might be better to make this number a constant?

Reply:
ok

Reply:
what is that number?

Reply:
// This balance is about right with 1 u ingested 1 * effect it about does one damage